.github/workflows/lint.yml

```diff
@@ -8,14 +8,14 @@ on:
       - stable*
 
 jobs:
-  php:
+  php-lint:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
+        php-versions: ['7.3', '7.4', '8']
```

Comment:
Regarding this I'm not sure, if it is necessary?
Are there large syntax changes on php, that could produce problems or why is linting done on multiple versions?

Reply:
Just do linting its a few seconds only.